pkg/commands/render.go

```diff
@@ -129,14 +129,16 @@ func makeRTEObjects(commonOpts *CommonOptions) ([]client.Object, string, error)
 		return nil, namespace, err
 	}
 
-	mf, err := rtemanifests.GetManifests(commonOpts.UserPlatform)
+	mf, err := rtemanifests.GetManifests(commonOpts.UserPlatform, namespace)
 	if err != nil {
 		return nil, namespace, err
 	}
 	mf = mf.Update(rtemanifests.UpdateOptions{
-		ConfigData:       commonOpts.RTEConfigData,
-		PullIfNotPresent: commonOpts.PullIfNotPresent,
-		Namespace:        namespace,
+		ConfigData: commonOpts.RTEConfigData,
+		DaemonSetOptions: &manifests.DaemonSetUpdateOptions{
```

Comment:
I'm really not sold regarding the need of a substruct. Maybe we can move back `PullIfNotPresent` in the `rtemanifests.UpdateOptions`.

Reply:
yee, I thought I will need additional parameters. Will revert it